roadmap/implementers-guide/src/runtime/scheduler.md

```diff
@@ -16,48 +16,64 @@ It aims to achieve these tasks with these goals in mind:
+}
 ```
 
-```text
-Availability Core Transitions within Block
-
-              +-----------+                |                    +-----------+
-              |           |                |                    |           |
-              | Free      |                |                    | Occupied  |
-              |           |                |                    |           |
-              +--/-----\--+                |                    +--/-----\--+
-               /-       -\                 |                     /-       -\
- No Backing  /-           \ Backing        |      Availability /-           \ No availability
-           /-              \               |                  /              \
-         /-                 -\             |                /-                -\
-  +-----v-----+         +----v------+      |         +-----v-----+        +-----v-----+
-  |           |         |           |      |         |           |        |           |
-  | Free      |         | Occupied  |      |         | Free      |        | Occupied  |
-  |           |         |           |      |         |           |        |           |
-  +-----------+         +-----------+      |         +-----|---\-+        +-----|-----+
-                                           |               |    \               |
-                                           |    No backing |     \ Backing      | (no change)
-                                           |               |      -\            |
-                                           |         +-----v-----+  \     +-----v-----+
-                                           |         |           |   \    |           |
-                                           |         | Free      -----+---> Occupied  |
-                                           |         |           |        |           |
-                                           |         +-----------+        +-----------+
-                                           |                 Availability Timeout
+```dot process
+digraph {
+	label = "Availability Core Transitions within Block\n\n\n";
+	labelloc = "t";
+	splines="line";
+
+	subgraph cluster_left {
+		label = "";
+		labelloc = "t";
+
+		fr1 [label = "Free" shape=rectangle]
+		fr2 [label = "Free" shape=rectangle]
+		occ [label = "Occupied" shape=rectangle]
+
+		fr1 -> fr2 [label = "No Backing"]
+		fr1 -> occ [label = "Backing"]
+
+		{ rank=same fr2 occ }
+	}
+
+	subgraph cluster_right {
+		label = "";
+		labelloc = "t";
+
+		occ2 [label = "Occupied" shape=rectangle]
+		fr3 [label = "Free" shape=rectangle]
+		fr4 [label = "Free" shape=rectangle]
+		occ3 [label = "Occupied" shape=rectangle]
+		occ4 [label = "Occupied" shape=rectangle]
+
+		occ2 -> fr3 [label = "Availability"]
+		occ2 -> occ3 [label = "No availability"]
+		fr3 -> fr4 [label = "No backing"]
+		fr3 -> occ4 [label = "Backing"]
+		occ3 -> occ4 [label = "(no change)"]
+		fr4 -> occ4 [label = "Availability Timeout"]
```

Comment:
I think that arrow was accidentally backwards in the old diagram, but it should be the other way around.

Reply:
And actually i think it makes most sense to change it to `occ3 -> fr3` now that i look closer